docs-src/docs/app.md

```diff
@@ -39,24 +39,25 @@ Parameters in _italics_ will still work but are **deprecated** - this means you
-| repeat(time, command, args=None) | time (int), command (function name), args (list of arguments) | -       | Repeats `command` every `time` milliseconds. This is useful for scheduling a function to be regularly called, for example updating a value read from a sensor. |
-| show()                           | -                                                             | -       | Displays the app window if it was previously hidden                                                                                                            |
-| update()                         | -                                                             | -       | Force the application to update itself, useful if changes aren't reflected in the UI.                                                                          |
-| set_full_screen(keybind)         | String                                                        | -       | Set the application to display full screen. Option to specify a key to exit full screen (default is 'Esc'.)                                                    |
-| _set_title(title)_               | _title (string)_                                              | -       | _Replaced by `title` property_                                                                                                                                 |
-| _bgcolor_                        | _bgcolor (string)_                                            | -       | _Replaced by `bg` property_                                                                                                                                    |
+| Method                                                                                               | Takes                                                                                               | Returns | Description                                                                                                                                                    |
+|------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| add_tk_widget(tk_widget, grid=None, align=None, visible=True, enabled=None, width=None, height=None) | tk_widget (tk), grid (list), align (str), visible (bool), enabled (bool), width (int), height (int) | Widget  | Adds a tk widget into a guizero container.)                                                                                                                    |
```

Comment:
Possibly add a note to say that this is for advanced users, as this appears alphabetically first in the list and may put people off?

Reply:
Changed the description to be 

`Adds a tk widget into a guizero container. Note - this is an advanced feature see [Using tk](usingtk.md) for more information.`